app/initial_data.py

```diff
@@ -102,6 +40,6 @@ def wait_for_app():
         wait_for_app()
 
     db = SessionLocal()
-    populate_initial_data(db)
+    populate_initial_data(db, get_password_hash)
```

Comment:
People may have a view if the `security` package should end up in `db-client` - doesn't quite feel right to me - but happy to be convinced otherwise.

Reply:
I'd agree that it doesnt quite seem right